packages/graphql-codegen-cli/src/loaders/schema/schema-from-string.ts

```diff
@@ -1,11 +1,10 @@
 import { Types } from 'graphql-codegen-core';
 import { SchemaLoader } from './schema-loader';
-import isValidPath = require('is-valid-path');
 import { parse, DocumentNode } from 'graphql';
 
 export class SchemaFromString implements SchemaLoader {
   canHandle(str: string): boolean {
-    if (isValidPath(str)) {
+    if (/\.[a-z0-9]+$/i.test(str)) {
```

Comment:
should it be changed on other `is-valid-path` usages?
maybe also adding a comment about the regex?

Reply:
@Urigo Now it has a comment.

About `is-valid-path` makes sense for other loaders because it still makes sure the path is valid, same with `is-glob`. They both just don't work well with SDL  (`scalar Date` passes because it might be a file / directory).